debian/marquee/03-nginx-conf

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
```

Comment:
Can you delete all the apache files and renumber these? They're in git history if we want to go back. certbot will need some tweaking. On bgsound I have `mkdir -p /var/www/http/.well-known/acme-challenge/` in 00-nginx-conf and `certbot certonly -n --agree-tos --webroot -m admin@whatwg.org -d $domain -w /var/www/http` in 01-certbot. There will be a TODO about reloading nginx when new certs are issued too.